### PR TITLE
content: create new website article

### PR DESCRIPTION
**Pull request by Barbapapazes**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 📰 Content (a new article or a change in the content folder)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Need to infuse your projects with UnJS aesthetics? Our design kit is your go-to resource, offering easy access to our colors, logos, and package visuals. If you do something cool with it, please share it with us [on X](https://x.com/unjsio)!
+
+![Screenshot of the design kit](/assets/images/blog/2023-12-11-a-new-home-to-drive-a-vision/design-kit.webp)
+
+Additionally, we also build internal and external tools to help you and us understand and manage the ecosystem. We will share some of them with you in the future.
+
+## Coming in 2024
+
+- presentation of what UnJS is
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:54]:
```suggestion
```

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- the 3 types of content
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:55]:
```suggestion
```

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- monthly updates
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:56]:
```suggestion
```

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+- more stats on the evolution of UnJS
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:57]:
```suggestion
```

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:58]:
```suggestion
```

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+authors:
+  - name: Estéban S
+    picture: https://github.com/barbapapazes.png
+    twitter: soubiran_
+categories:
+  - website
+packages:
+publishedAt: 2023-12-11
+modifiedAt: 2023-12-11
+layout: blog-post # TODO: remove
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:13]:
```suggestion
```

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+title: A New Home to Drive a Vision
+description: Deep dive into the new theme for the UnJS website and discover what's coming in 2024
+authors:
+  - name: Estéban S
+    picture: https://github.com/barbapapazes.png
+    twitter: soubiran_
+categories:
+  - website
+packages:
+publishedAt: 2023-12-11
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:11]:
```suggestion
publishedAt: 2023-12-30
```

File: content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+modifiedAt: 2023-12-11
```
Comment (Barbapapazes) [inline content/5.blog/2023-12-11-a-new-home-to-drive-a-vision.md:12]:
```suggestion
modifiedAt: 2023-12-30
```

**Review by pi0 — APPROVED** [review]
💛

**Pull request merged**